src/main/java/factionsplusplus/models/PollVote.java

```diff
@@ -0,0 +1,69 @@
+package factionsplusplus.models;
```

Comment:
I wouldn't add dependencies to subclasses, I would just make the top most layer (Poll) have whatever dependencies and handle updates for the lower classes